website/content/docs/install-boundary/install-clients.mdx

```diff
@@ -0,0 +1,61 @@
+---
+layout: docs
+page_title: Install Boundary clients
+description: >-
+ Install the Boundary CLI, Boundary Desktop Client app, or Boundary Installer for Boundary Enterprise or Community Edition on Windows, MacOS, or Linux.
+---
+
+# Install Boundary clients
+
+Boundary end users access infrastructure targets using the Boundary Desktop Client app or the Boundary CLI. Enterprise or HCP Boundary users can use transparent sessions to proxy traffic though the Boundary Client Agent <sup>HCP/ENT</sup>.
+
+This guide outlines the required steps to manually install and configure a Boundary client depending on which distribution of Boundary end users access.
+
+To learn about installing the Boundary binary when you deploy Boundary controllers or workers, refer to the [Deploy Boundary](/boundary/docs/install-boundary/deploy) page.
+
+Select your Boundary distribution, and complete the steps to install the
+correct clients:
+
+<Tabs>
+<Tab heading="Enterprise">
+
+@include 'alerts/enterprise-only.mdx'
+
+@include 'alerts/beta.mdx'
```

Comment:
This warning just alludes to the fact that the entire Desktop client for Enterprise is beta? Should it clarify that the TS portion is beta? Or maybe suggest the version to use if they want to use beta feature of TS?

Reply:
hmm yeah I'm not sure. I think since the client agent is beta, and it's bundled with the installer, that would inherently make the entire installer itself beta. but I'm not sure. Considering this warning will go away with GA, I'm not sure how much to worry about it

Reply:
Yeah, maybe not that big of a deal!

Reply:
I added it as a note instead, what do you think?